scripts/airdrop.remove-principals.sh

```diff
@@ -0,0 +1,46 @@
+#!/usr/bin/env bash
+
+# Get list of principals to remove from the airdrop
+
+set -eExuo pipefail
+
+principals=()
+
+print_usage_and_exit() {
+    echo "Error: $1"
+    echo "Usage: $0 --principals=value1,value2"
+    exit 1
+}
+
+# Parse named arguments
+while [ "$#" -gt 0 ]; do
+    case "$1" in
+        --principals=*)
+            IFS=',' read -ra principals <<< "${1#*=}"
+            ;;
+        *)
+            print_usage_and_exit "Invalid argument - $1"
+            ;;
+    esac
+    shift
+done
+
+case $ENV in
```

Comment:
we have this now in multiple scripts, not a prio but if one of these days you have time to refactor it, would be neat

Reply:
yeah, most the scripts by now have a lot of commonality

Reply:
I agree - I can take a look tomorrow - also we should read the wallet from file ideally!